taskiq_redis/redis_cluster_broker.py

```diff
@@ -0,0 +1,77 @@
+from typing import Any, AsyncGenerator, Callable, Optional, TypeVar
+
+from redis.asyncio import RedisCluster
+from taskiq.abc.broker import AsyncBroker
+from taskiq.abc.result_backend import AsyncResultBackend
+from taskiq.message import BrokerMessage
+
+_T = TypeVar("_T")
+
+
+class BaseRedisClusterBroker(AsyncBroker):
+    """Base broker that works with Redis Cluster."""
+
+    def __init__(
+        self,
+        url: str,
+        task_id_generator: Optional[Callable[[], str]] = None,
```

Comment:
Let's remove id generator from here. We have new builder API to set generator and result backend.

Reply:
Done